cookbook/specs/near.json

```diff
@@ -492,6 +501,19 @@
                             },
                             {
                                 "name": "pruning",
```

Comment:
let's do this also pruning-archive, and then it inherits the 
"pruning" verification for a regular node

Reply:
https://github.com/lavanet/lava/pull/1498/commits/992a1b3a1ca3b7c3de412acfc59099e95eaa0231

cahnged naming and it is working tested on my nodes.